utils/utils.c

```diff
@@ -237,6 +237,21 @@ glusterBlockLogdirCreate(void)
 }
 
 
+void fetchGlfsVolServerFromEnv()
+{
+  char *volServer;
+
+
+  volServer = getenv("GD2_ENDPOINTS");
```

Comment:
@aravindavk good point.

@amarts pointed this name from https://github.com/gluster/glusterd2/pull/1292/files (assuming other projects prefer it right now) 

But I agree, gluster-blockd can use a different name. And it is the deployers/installers/admins duty to fetch it from environment and set it in the config-file.

How about GB_BHV_VOLSERVER (one address only so no plural) as all the other options in configfile start with GB_XYZ or I'm okay with anything which start with GB_

BHV -> block hosting volume

Thanks!

Reply:
any plans to support multiple vol servers? since this stays outside the cluster it can connect to any node of cluster.

`GB_BHV_VOLSERVER` looks good to me.

GCS deploy can set env variables during deployment(Example for Glusterd2 pod: https://github.com/gluster/gcs/pull/43)

Reply:
> any plans to support multiple vol servers? since this stays outside the cluster it can connect to any node of cluster.

For GCS & Kube the expectation is, we will have a service which will take care of resolving the service (fqdn) to backend volfile servers hostnames/Ip's, which can be at different/same namespaces.

If that is the case why do we need multiple vol servers ?  

@amarts correct me.

> 
> `GB_BHV_VOLSERVER` looks good to me.

cool, will update the existing with GB_BHV_VOLSERVER.

> 
> GCS deploy can set env variables during deployment(Example for Glusterd2 pod: [gluster/gcs#43](https://github.com/gluster/gcs/pull/43))

Awesome. Thanks!